File: src/main/kotlin/com/hxl/blog/controller/ip/entity/TbIp.java
package com.hxl.blog.controller.ip.entity;

import com.baomidou.mybatisplus.annotation.IdType;
import com.baomidou.mybatisplus.annotation.TableId;
import com.baomidou.mybatisplus.annotation.TableName;
import java.io.Serializable;
import java.time.LocalDateTime;

/**
 * <p>
 * 
 * </p>
 *
 * @author hxl
 * @since 2021-11-12
 */
@TableName("tb_ip")
public class TbIp implements Serializable {

    private static final long serialVersionUID = 1L;

    @TableId(value = "id", type = IdType.AUTO)
    private Integer id;

    private String ipAddress;

    private String ipCity;

    private String ipProvince;

    private LocalDateTime createDate;

    public Integer getId() {
        return id;
    }

    public void setId(Integer id) {
        this.id = id;
    }
    public String getIpAddress() {
        return ipAddress;
    }

    public void setIpAddress(String ipAddress) {
        this.ipAddress = ipAddress;
    }
    public String getIpCity() {
        return ipCity;
    }

    public void setIpCity(String ipCity) {
        this.ipCity = ipCity;
    }
    public String getIpProvince() {
        return ipProvince;
    }

    public void setIpProvince(String ipProvince) {
        this.ipProvince = ipProvince;
    }
    public LocalDateTime getCreateDate() {
        return createDate;
    }

    public void setCreateDate(LocalDateTime createDate) {
        this.createDate = createDate;
    }

    @Override
    public String toString() {
        return "TbIp{" +
            "id=" + id +
            ", ipAddress=" + ipAddress +
            ", ipCity=" + ipCity +
            ", ipProvince=" + ipProvince +
            ", createDate=" + createDate +
        "}";
    }
}

File: src/main/kotlin/com/hxl/blog/controller/ip/entity/TbIp.kt
package com.hxl.blog.controller.ip.entity

import com.baomidou.mybatisplus.extension.service.impl.ServiceImpl
import com.hxl.blog.controller.ip.mapper.TbIpMapper
import com.hxl.blog.controller.ip.entity.TbIp
import com.hxl.blog.controller.ip.service.ITbIpService
import com.baomidou.mybatisplus.extension.service.IService
import com.baomidou.mybatisplus.core.mapper.BaseMapper
import com.baomidou.mybatisplus.annotation.TableId
import com.baomidou.mybatisplus.annotation.IdType
import com.baomidou.mybatisplus.annotation.TableName
import org.springframework.web.bind.annotation.RequestMapping
import java.io.Serializable
import java.time.LocalDateTime

/**
 *
 *
 *
 *
 *
 * @author hxl
 * @since 2021-10-27
 */
@TableName("tb_ip")
class TbIp : Serializable {
    @TableId(value = "id", type = IdType.AUTO)
    var id: Int? = null
    var ipAddress: String? = null
    var ipCity: String? = null
    var ipProvince: String? = null
    var createDate: LocalDateTime? = null
    override fun toString(): String {
        return "TbIp{" +
                "id=" + id +
                ", ipAddress=" + ipAddress +
                ", ipCity=" + ipCity +
                ", ipProvince=" + ipProvince +
                "}"
    }

    companion object {
        private const val serialVersionUID = 1L
    }
}